config/core/resources/topic.yaml

```diff
@@ -46,10 +46,17 @@ spec:
     - name: Age
       type: date
       JSONPath: .metadata.creationTimestamp
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+  # All versions happen to have the same schema today. They will likely diverge in the future.
```

Comment:
I'd probably remove this and the other similar one. Can't guarantee it will be removed when it goes out of date.

Reply:
If they ever diverge, we will actually end up removing the `validation` field one line down and adding the individual schemas in the `versions` field. I tried to do that initially, but the CRD was rejected if the schema of all versions was identical and specified in the `versions` field.